.run/Deploy kyma.run.xml

```diff
@@ -1,6 +1,6 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Deploy kyma" type="ShConfigurationType">
-    <option name="SCRIPT_TEXT" value="echo Enter the skr host value \(host.k3d.internal or localhost\):; read SKR_HOST; export SKR_HOST; export KUBECONFIG=$HOME/.k3d/kcp-local.yaml; $PROJECT_DIR$/scripts/tests/deploy_kyma.sh" />
+    <option name="SCRIPT_TEXT" value="echo Enter the skr host value \(host.k3d.internal or localhost\):; read TEMP; $PROJECT_DIR$/scripts/tests/deploy_kyma.sh $TEMP" />
```

Comment:
Deploying kyma using this results in an error although I entered "localhost" when prompted

```
Enter the skr host value (host.k3d.internal or localhost):
localhost
Usage: /Users/I559973/Repos/lifecycle-manager/scripts/tests/deploy_kyma.sh [localhost | host.k3d.internal]
localhost: Use this option when running KLM locally on the machine
host.k3d.internal: Use this option when deploying KLM onto a cluster
```